styles/header.css

```diff
@@ -0,0 +1,172 @@
+a {
+  text-decoration: none;
+}
+
+.pj-logo {
+  border: 1px solid red;
+}
+
+/* Social Navigation */
+.social-nav {
+  background-color: #333;
+  color: #fff;
+  display: flex;
+  justify-content: end;
+}
+
+.social-nav a {
+  border: none;
+  color: #fff;
+  cursor: pointer;
+  padding: 10px;
+  display: inline-block;
+  font-size: 13px;
+}
+
+.social-nav ul {
+  display: flex;
+  list-style: none;
+}
+
+.nav-wrap {
+  width: 900px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: end;
+  padding: 0 20px;
+}
+
+/* Main Navigation */
+.main-nav {
+  display: flex;
+  padding: 20px 0;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2);
+}
+
+.main-nav ul {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  margin-right: 60px;
+}
+
+.main-nav ul a {
+  padding: 20px;
+  color: #999;
+}
+
+.main-nav ul a.selected {
+  color: red;
+}
+
+.main-nav ul a:hover {
+  color: red;
+}
+
+.main-nav-inner {
+  display: flex;
+  width: 900px;
+  justify-content: space-between;
+  margin: 0 auto;
+  align-items: center;
+  padding: 0 20px;
+}
+
+.right-side-menus {
+  display: flex;
+}
+
+.nav-button {
+  padding: 10px;
+  background-color: transparent;
+  border: 3px solid red;
+  color: red;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] The CSS styling of this project should be built as a mobile-first.
Please fix it, media queries are supposed to be used in this project to change the design from mobile to desktop, not the other way around.